lightning/src/ln/channel.rs

```diff
@@ -62,6 +62,17 @@ pub struct ChannelValueStat {
 	pub counterparty_dust_limit_msat: u64,
 }
 
+pub struct AvailableBalances {
+	/// The amount that would go to us if we close the channel, ignoring any on-chain fees.
+	pub balance_msat: u64,
+	/// Total amount available for our counterparty to send to us, ignoring HTLCs.
```

Comment:
Fwiw, I don't see how HTLCs are ignored? Seems pending htlcs are factored into these

Reply:
Ah...I copied it from the fn docs, indeed, that's somewhat nonsensical.